release-notes.txt

```diff
@@ -1,186 +1,51 @@
 BISDN Linux
-4.3.0 Release Notes
-Date: 2021-11-12
-
-! New in this release
-- BISDN Linux now ships with a default en-us locale, allowing applications
-  that rely on locales like tmux to work.
-- Changing a bridge VLAN's egress tagged or pvid state during runtime is now
-  properly handled.
-- Accton AS4610: Added support for the AS4610-54T (B) model with reversed
-  airflow. Special thanks to Ralf Kundel for helping with testing the
-  compatibility.
+4.3.1 Release Notes
+Date: 2021-11-30
 
 ! Changed behaviour
-- Accton AS4610: Since we now respect the Bootloader's passed memory limits,
-  the available memory is reduced by 16 MiB.
+- Certain utilities like client_drivshell will now require root privileges
 
 ! Bug fixes
-- Fixed STP not properly working if the untagged default VLAN is not 1.
-- Fixed a race when applying MSTP states.
-- Fixed calculating effective VLAN STP states on STP updates.
-- Fixed baseboxd crashing when adding custom fdb entries.
-- Fixed a check that caused a routing entry for the IPv6 lo address to be
-  created in some situations.
-- Fixed ip(6)tables rules sometimes not being applied during boot.
-- Properly handle clients switching ports and update their forwarding rules.
-- Accton AS4610: Fixed the second CPU sometimes not coming up.
-- Accton AS4610: Properly respect U-Boot's passed memory limits.
-- Accton AS4610: 1G LEDs blink for traffic again.
+- Validate stp arguments in ofdpa-grpc instead of passing them on as is
+
+! Security fixes
+- Fixed an issue where unauthenticated users could inject shell commands
+  executed with root privileges over the network
+- Fixed an issue where local users could inject shell commands executed
+   with root privileges
```

Comment:
```suggestion
- Fixed issues where users could inject shell commands either locally or unauthenticated over the network 
   being executed with root privileges
```

Reply:
Not sure if it makes sense merging these two distinct issues. If we do, I would at least move the executed with root privileges to the shell commands, and maybe s/users/actors/.
e.g.
```suggestion
- Fixed issues allowing actors to inject shell commands executed with root privileges locally and unauthenticated over the network.
```

Reply:
sounds good